client/common/sass/components/_card-image.sass

```diff
@@ -1,8 +1,10 @@
 .card-image
 	&-big
+		max-width: 100%
```

Comment:
I think this is repeated code from below, since I needed to add the same lines in blog index page 😅 Can be removed.